release-notes.md

```diff
@@ -9,7 +9,27 @@ meta:
 ---
 # Release Notes
 
+## 2.3.0
+### Admission Controller
+We rolled out a new UI that will let you see what’s going on with your Admission Controller. If a resource is rejected from your cluster, you will now see all of the action items associated with that event, along with a remediation recommendation. 
+ 
+### More Costs Visualizations
+ We’ve added more visualizations for Memory Usage and CPU usage.
```

Comment:
I think this is referring to the prometheus stuff. Maybe we want to label it as a closed beta? We have the gathering piece in and it'll show more accurate costs, the visualization piece didn't make it in this time.

Reply:
Will the charts go out in the next release (in January)?

Reply:
I think we can remove this one. It'll go live in Jan